src/components/ConfigureNewVault.tsx

```diff
@@ -0,0 +1,25 @@
+import AmountInput from 'components/AmountInput';
+
+const ConfigureNewVault = () => {
+  return (
+    <div className="mt-8 px-12 py-8 bg-white rounded-[20px] shadow-[0_40px_40px_0_rgba(116,116,116,0.25)]">
+      <h3 className="mb-3 font-serif font-bold leading-[26px]">Configure</h3>
+      <p className="font-serif text-[#666980] leading-[26px]">
+        Choose your vault parameters.
+      </p>
+      <div className="mt-12 flex gap-x-20 gap-y-6 flex-wrap">
+        <AmountInput
+          label="Atom to lock up *"
+          error="Need to obtain funds"
+        ></AmountInput>
+        <AmountInput label="Collateralization ratio *"></AmountInput>
```

Comment:
AmountInput doesn't take `children`, right?
```suggestion
        <AmountInput label="Collateralization ratio *" />
```

Reply:
Yea done